src/simulators/tensor_network/matrix_product_state_tensor.hpp

```diff
@@ -124,211 +135,197 @@ class MPS_Tensor
-	{
-		if (data_.size() != 2)
-		{
-			cout << "ERROR: The tensor doesn't represent one qubit" << '\n';
-			assert(false);
-		}
-		data_[1] = data_[1] * complex_t(0, -1);
-	}
-	void MPS_Tensor::apply_t()
-	{
-		if (data_.size() != 2)
-		{
-			cout << "ERROR: The tensor doesn't represent one qubit" << '\n';
-			assert(false);
-		}
-		data_[1] = data_[1] * complex_t(SQR_HALF, SQR_HALF);
-	}
-	void MPS_Tensor::apply_tdg()
-	{
-		if (data_.size() != 2)
-		{
-			cout << "ERROR: The tensor doesn't represent one qubit" << '\n';
-			assert(false);
-		}
-		data_[1] = data_[1] * complex_t(SQR_HALF, -SQR_HALF);
-	}
+void MPS_Tensor::apply_tdg()
+{
+  if (data_.size() != 2)
+    {
+      cout << "ERROR: The tensor doesn't represent one qubit" << '\n';
+      assert(false);
+    }
+  data_[1] = data_[1] * complex_t(SQR_HALF, -SQR_HALF);
+}
 
-	void MPS_Tensor::apply_matrix(cmatrix_t &mat)
-	{
-		if (data_.size() != mat.GetRows())
-		{
-			cout << "ERROR: The matrix and tensor doesn't represent the same number of qubits" << '\n';
-			assert(false);
-		}
-
-		cvector_t temp;
-		for (uint_t a1 = 0; a1 < data_[0].GetRows(); a1++)
-			for (uint_t a2 = 0; a2 < data_[0].GetColumns(); a2++)
-			{
-				temp = get_data(a1,a2);
-				temp = mat * temp;
-				insert_data(a1,a2,temp);
-			}
-	}
-	void MPS_Tensor::apply_cnot(bool swapped)
-	{
-		if (data_.size() != 4)
-		{
-			cout << "ERROR: The tensor doesn't represent 2 qubits" << '\n';
-			assert(false);
-		}
-		if(!swapped)
-			swap(data_[2],data_[3]);
-		else
-			swap(data_[1],data_[3]);
-	}
+void MPS_Tensor::apply_matrix(cmatrix_t &mat)
+{
+  if (data_.size() != mat.GetRows())
+    {
+      cout << "ERROR: The matrix and tensor doesn't represent the same number of qubits" << '\n';
+      assert(false);
+    }
 
-	void MPS_Tensor::apply_swap()
-	{
-		if (data_.size() != 4)
-		{
-			cout << "ERROR: The tensor doesn't represent 2 qubits" << '\n';
-			assert(false);
-		}
-		swap(data_[1],data_[2]);
-	}
-	void MPS_Tensor::apply_cz()
-	{
-		if (data_.size() != 4)
-		{
-			cout << "ERROR: The tensor doesn't represent 2 qubits" << '\n';
-			assert(false);
-		}
-		data_[3] = data_[3] * (-1.0);
-	}
+  cvector_t temp;
+  for (uint_t a1 = 0; a1 < data_[0].GetRows(); a1++)
+    for (uint_t a2 = 0; a2 < data_[0].GetColumns(); a2++)
+      {
+	temp = get_data(a1,a2);
+	temp = mat * temp;
+	insert_data(a1,a2,temp);
+      }
+}
+
+void MPS_Tensor::apply_cnot(bool swapped)
+{
+  if (data_.size() != 4)
+    {
+      cout << "ERROR: The tensor doesn't represent 2 qubits" << '\n';
+      assert(false);
+    }
+  if(!swapped)
+    swap(data_[2],data_[3]);
+  else
+    swap(data_[1],data_[3]);
+}
 
+void MPS_Tensor::apply_swap()
+{
+  if (data_.size() != 4)
+    {
+      cout << "ERROR: The tensor doesn't represent 2 qubits" << '\n';
+      assert(false);
+    }
+  swap(data_[1],data_[2]);
+}
 
+void MPS_Tensor::apply_cz()
+{
+  if (data_.size() != 4)
+    {
+      cout << "ERROR: The tensor doesn't represent 2 qubits" << '\n';
```

Comment:
Remove `cout`

Reply:
Removed this error - it was for our development phase.